rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -0,0 +1,1336 @@
+        required: false
+    output:
+      id:
+        description: Identifier of the resource created
+        title: Scan ID
+        type: integer
+        example: ' {
+              "scans": [
+                {
+                  "asset_ids": [
+                    "cdc978de-f683-4178-a1d9-d5a94a114b87-default-asset-13"
+                  ],
+                  "engine_id": "c8970d2ee174180e202af83b66cd0c4b",
+                  "finished": null,
+                  "id": "7fc3d8ec-24a7-4835-84f2-8877fe96a29b",
+                  "name": "testing scan action"
+                }
+              ],
+              "unscanned_assets": []
+            }'
+        required: false
+  get_scan:
+    title: Get Scan
+    description: Get the status of a scan
+    input:
+      scan_id:
+        title: Scan ID
+        description: ID of the scan to obtain
+        type: string
+        example: "abb37782-df95-4cf6-b4c2-8d466ca57349"
+        required: true
+    output:
+      asset_ids:
+        description: List of IDs of the scanned assets
+        title: Asset IDs
+        type: "[]string"
+        example: ["cdc978de-f683-4178-a1d9-d5a94a114b87-default-asset-13"]
+        required: false
+      engine_id:
+        description: ID of the engine used for the scan
+        title: Engine ID
+        type: string
+        example: "c8970d2ee174180e202af83b66cd0c4b"
+        required: false
+      scanName:
+        description: User-driven scan name for the scan
+        title: Scan Name
+        type: string
+        example: "testing scan action"
+        required: false
+  asset_search:
+    title: Asset Search
+    description: Search for assets using filtered asset search
+    input:
+      size:
+        title: Size
+        description: The number of records to retrieve. If blank or '0' all assets that match the search will be returned
+        type: number
+        required: false
+        default: 0
+        example: 100
+      pages:
+        title: Pages
+        description: The number of pages of records to retrieve
+        type: number
+        required: false
+        default: 1
+        example: 100
+      sort_criteria:
+        title: Sort Criteria
+        description: JSON object for sorting by criteria. Multiple criteria can be specified with an order of 'asc' (ascending) or 'desc' (descending)
+        type: object
+        required: false
+        example: '{"risk-score": "asc", "criticality-tag": "desc"}'
+      ip:
+        title: IP
+        description: Primary IPv4 or IPv6 address of the asset
+        type: "[]string"
+        example: [ "10.4.31.141" ]
+        required: false
+      hostname:
+        title: Hostname
+        description: The hostname
+        type: "[]string"
+        example: [ "fortigate-vm02.vuln.lax.rapid7.com" ]
+        required: false
+    output:
+      assets:
+        title: Assets
+        description: List of asset details returned by the search
+        type: '[]search_asset'
```

Comment:
Return `[]assets` here?

Reply:
asset and sear_asset are two different things